src/main/java/net/greatstart/dto/DtoProjectDescription.java

```diff
@@ -21,6 +21,7 @@
     private BigDecimal minInvestment;
     private LocalDate dateStart;
     private LocalDate dateAdded;
+    private String other;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "other" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)